shell.nix

```diff
@@ -1,10 +1,4 @@
-{ pkgs ? import <nixpkgs> {} }:
+{ pkgs ? import <nixpkgs> { } }:
 pkgs.mkShell {
-  nativeBuildInputs = with pkgs; [
-    opam
-    pkg-config
-    gmp
-    openssl
-    libev
-  ];
+  nativeBuildInputs = with pkgs; [ pkg-config gmp libffi libev sqlite openssl ];
```

Comment:
Don't we still need `opam`?

Reply:
I've been using dune package management when working on this. Also since opam has a user-wide configuration I tend to use a user-wide installation of opam (e.g. with home-manager) rather than installing it with a project's nix config. That way updates to the opam version used by a project's nix config won't affect my user-wide opam configuration (this happened in the dune repo recently and it was annoying to deal with).